app/controllers/api/v1/collaterals_kind_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Api
+  module V1
+    class CollateralsKindController < ApplicationController
+      def index
+        render json: CollateralsKind::IndexService.call
+      end
+
+      def create
+        data = params.require(:data).permit(attributes: [:name, :color])['attributes']
+        dto = CollateralsKind::CreateCollateralKindDto.new(data['name'], data['color'])
+
+        render json: CollateralsKind::CreateService.call(dto)
```

Comment:
how about passing a data var into DTO, and make destructuring inside DTO class? In case when no. of args will grow, this generates a lot of code.

Example
```
CollateralsKind::CreateCollateralKindDto.new(data)
```